jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecr/actions/PushToRepositoryAction.kt

```diff
@@ -203,54 +203,47 @@ internal class PushToEcrDialog(
 
     private fun localImageSelectorPanel() = panel {
         row(message("ecr.push.source")) {
-            // property binding syntax causes kotlin compiler error for some reason
             comboBox(
                 localImageRepoTags,
-                { localImage },
-                { ::localImage.set(it) },
                 listCellRenderer { value, _, _ ->
                     text = value.tag ?: value.imageId.take(15)
                 }
-            )
+            ).bindItem(::localImage.toNullableProperty())
                 .applyToComponent { ComboboxSpeedSearch(this) }
-                .growPolicy(GrowPolicy.MEDIUM_TEXT)
-                .withErrorOnApplyIf(message("ecr.image.not_selected")) { it.selected() == null }
+                .errorOnApply(message("ecr.image.not_selected")) { it.selected() == null }
+                .resizableColumn()
+                .columns(30) // The size of the entire dialog is doubling if specific columns are not set for this component
```

Comment:
unsure why we need resizableColumn if the width is fixed

Reply:
removed it